contracts/KyberIEOWrapper.sol

```diff
@@ -1,7 +1,57 @@
-pragma solidity ^0.4.0;
+pragma solidity ^0.4.23;
 
-contract KyberIEOWrapper {
-    function KyberIEOWrapper(){
 
+import './KyberIEO.sol';
+import './ERC20Interface.sol';
+
+
+interface KyberNetwork {
+    function trade(ERC20 src, uint srcAmount, ERC20 dest, address destAddress, uint maxDestAmount,
+        uint minConversionRate, address walletId) external payable returns(uint);
+    function getExpectedRate(ERC20 src, ERC20 dest, uint srcQty) external view
+        returns (uint expectedRate, uint slippageRate);
+}
+
+
+contract KyberIEOWrapper is Withdrawable {
+
+    KyberNetwork kyberNetwork;
+    KyberIEO kyberIeo;
+    ERC20 constant internal ETH_TOKEN_ADDRESS = ERC20(0x00eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee);
+
+    constructor(KyberNetwork _kyber, address _admin) public Withdrawable(_admin) {
```

Comment:
we agreed network address is passed as a param every time user want to contribute

Reply:
don't recall.
why as a param?